docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: "3.9"
+
+services:
+  db:
+    image: postgres:14.2
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+  web:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    volumes:
+      - .:/smartflix
+    ports:
+      - "3000:3000"
```

Just another detail, but the port can be a number instead of string 💅🏻 

```suggestion
      - 3000:3000
```